### DISCUSSION TITLE
Enable mocha-junit-reporter in a custom runner

### DISCUSSION THREAD

**Discussion by Todaichi** [codeceptjs/CodeceptJS#3781] [Q&A]
So, when running commands with codeceptjs you can pass the --reporter flag and then the type:
  **codeceptjs run  --reporter mocha-junit-reporter --steps**

I have had to make a custom runner for some out of the box thinking, which works great.  The last thing I need is to be able to tell the script that it is to use the reporter.   The config file is setup with the reporter info I want, but that doesn't actually say use it in the execution.   I am assuming that it is going to be in this line, in the options.
    **const codecept = new Codecept(codeceptConf.config, options);**
But, I don't know the format needed or what its actually looking at.  for example the flag --steps is changed to step:true in the options.

Thank you,

     --Daniel

**Comment by adudyak** [comment]
Kinda strange that you ask for format required for your own custom runner.
Here is a sample, how report is executed in GitHub Actions:
` 33       run: npx codeceptjs run --reporter mochawesome`
https://github.com/drozdik29/drozdovsky-js/blob/main/github/workflows/drozd_codeceptjs.yml
